programs/client/Client.cpp

```diff
@@ -1111,6 +1111,44 @@ void Client::readArguments(
     {
         std::string_view arg = argv[arg_num];
 
+        /// Support arguments with spaces around equals sign.
+        std::string fused_arg;
+        if (arg.starts_with('-') && arg_num + 1 < argc)
+        {
+            if (arg.find('=') == std::string::npos)
+            {
+                std::string_view next_arg = argv[arg_num + 1];
+                if (next_arg.starts_with('='))
+                {
+                    fused_arg = std::string(arg);
+                
+                    // Case: --param =value
```

Comment:
A little bit concerned about this case, as string-valued parameters can have values starting with `=`. Let's allow only when there are spaces at both sides.